executor/builder.go

```diff
@@ -422,10 +422,19 @@ func (b *executorBuilder) buildShowDDL(v *plannercore.ShowDDL) Executor {
 }
 
 func (b *executorBuilder) buildShowDDLJobs(v *plannercore.PhysicalShowDDLJobs) Executor {
+	loc := b.ctx.GetSessionVars().TimeZone
```

Comment:
Using getTimeZone() is enough?

Reply:
Yes, if it were exported :)

Do you want me to move the `getTimeZone()` function to `util/timeutil/time.go` and export it from there instead?

Reply:
I used the SessionVars `Location()` function instead, which is similar.